.env.dist

```diff
@@ -0,0 +1,2 @@
+UID=1000
```

Comment:
Is there not a better way to mount the dev directory without needing uid/gid? Not only this adds to the clutter but it also feels a bit wonky.

Reply:
Unfortunately, this is a years-long issue with docker and volumes - https://github.com/moby/moby/issues/2259. And in this case we need the executor of `npm run dev` to match the owner of the mounted directory, which can be anything. This solutions is one of the least complicated and working.